ddtrace/internal/writer.py

```diff
@@ -393,6 +396,8 @@ def _downgrade(self, payload, response):
     def _send_payload(self, payload, count):
         headers = self._headers.copy()
         headers["X-Datadog-Trace-Count"] = str(count)
+        if self._compute_stats:
+            headers["Datadog-Client-Computed-Stats"] = "yes"
```

Comment:
we can move this to `__init__` since `self._compute_stats` only gets passed in/changed from `__init__`